test/CodeGen/AVR/branch-relaxation-long.ll

```diff
@@ -0,0 +1,4133 @@
+; RUN: llc < %s -march=avr | FileCheck %s
+
+; CHECK-LABEL: relax_to_jmp
+; CHECK: cpi     r{{[0-9]+}}, 0
+; CHECK: brne    LBB0_1
+; CHECK: jmp     LBB0_2
+
+define i8 @relax_to_jmp(i1 %a) {
+entry-block:
```

Comment:
Add `CHECK-LABEL: LBB0_1` above this label.

If somehow the basic block default name numbering changed, this will catch that